src/screens/LocationPage/LocationRouter.tsx

```diff
@@ -2,19 +2,37 @@ import React from 'react';
 import { useParams, Redirect } from 'react-router-dom';
 import regions, { RegionContext } from 'common/regions';
 import LocationPage from './LocationPage';
+import RegionPage from './RegionPage';
 
 interface LocationPageUrlParams {
-  stateId: string;
+  stateId: string | undefined;
   countyId: string | undefined;
+  metroAreaUrlSegment: string | undefined;
 }
 
 const LocationRouter: React.FC = () => {
-  let { stateId, countyId } = useParams<LocationPageUrlParams>();
+  let { stateId, countyId, metroAreaUrlSegment } = useParams<
```

Comment:
I would do:
```
let region = useRegionFromParams();
```
and update the hook to query the new function you used.  then you can do:
```
if (region instance of MetroArea) {
 
} else {
return (
  <RegionContext.Provider value={metroArea}>
    <RegionPage region={metroArea} />
  </RegionContext.Provider>
);
```